assets/app/index.html

```diff
@@ -149,6 +149,7 @@
         <script src="scripts/services/ws.js"></script>
         <script src="scripts/services/userstore.js"></script>
         <script src="scripts/services/auth.js"></script>
+        <script src="scripts/services/apiService.js"></script>
```

Comment:
don't suffix a file under "services" with "Service", it's redundantly repetitive